Lib/test/test_typing.py

```diff
@@ -528,6 +528,7 @@ def test_repr(self):
         self.assertEqual(repr(Literal[int]), "typing.Literal[int]")
         self.assertEqual(repr(Literal), "typing.Literal")
         self.assertEqual(repr(Literal[None]), "typing.Literal[None]")
+        self.assertEqual(repr(Literal[1, 2, 3, 3]), "typing.Literal[1, 2, 3]")
```

Comment:
Are there checks verifying the value of `__args__`? Should it be `(1, 2, 3)` or `(1, 2, 3, 3)`? I can't easily follow the path through the code that would take. :-(